Marsiple/Comment/CommentViewController.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  CommentViewController.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 23/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import SnapKit
+
+class CommentViewController: UIViewController {
+    private let commentView = CommentView.autolayoutView()
+    
+    init() {
+        super.init(nibName: nil, bundle: nil)
+        setupView()
+        setupNavigationBar()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+private extension CommentViewController {
+    @objc func saveButtonTapped() { // TODO: Implement posting comment on-line
```

Comment:
As a convention, lets put `TODO`/`NOTE`/`FIXME` markers above the code they refer to, if there's space. If not, beside is completely fine. I'd say putting the marker beside is okay for markers with shorter messages.

In this case, we could put the `TODO` above the method definition. If you feel strongly about having the marker beside the code, I'm okay with that.